profile-widget/profilewidget2.cpp

```diff
@@ -796,6 +796,9 @@ void ProfileWidget2::plotDive(const struct dive *d, bool force, bool doClearPict
 		item->setModel(dataModel);
 		item->setEvent(event, lastgasmix);
 		item->setZValue(2);
+#ifndef SUBSURFACE_MOBILE
+		item->setScale(printMode ? 4 :1);
```

Comment:
should this be a fixed scale, or is there a reasonable way to calculate a "good" scale?
I'm not trying to make this harder for you, I'm really just asking

Reply:
I did a bit trial and error and 4 is about right for me. But of course this depends on screen resolution and your target resolution for the image file. Prescribing a fixed image size and adopting the scale factor accordingly does not really work, since the profile has the same aspect ration as on the screen (which is variable). One could try to be clever about this (like fixing either target width or height). What do you think?

Reply:
why do we have to set printMode on mobile?

Reply:
i think what will happen here is that for the profile image export functionality (divelogexportdialog.cpp # exportProfile()) the scale will end up being adjusted properly, given the QImage scales x4 that is also in this diff.

but for this functionality:
https://github.com/subsurface/subsurface/blob/5b078539a74ad65bd8e984bdb6f692ae4a2ae6e4/desktop-widgets/printer.cpp#L44

which renders the profile without the x4 scale, the above change will make event icons look big on the profiles that are printed via the file -> print func.